mrmustard/lab/abstract/transformation.py

```diff
@@ -386,6 +386,26 @@ def __eq__(self, other):
         return True
 
     def __repr__(self):
+
+        class_name = self.__class__.__name__
+        modes = self._modes
```

Comment:
isn't it better to use the property `self.modes`?

Reply:
Yes!